README.md

```diff
@@ -112,3 +121,138 @@ make
 # T2 Hardware API
 
 When you `require('tessel')` within a script which is executed on Tessel 2, this loads a library which interfaces with the Tessel 2 hardware, including pins, ports, and LEDs, just like Tessel 1 ([Tessel 1 hardware documentation](https://tessel.io/docs/hardwareAPI)). The code for Tessel 2's hardware object can be found [here](https://github.com/tessel/t2-firmware/blob/master/node/tessel.js).
+
+## Ports and pins
+
+Tessel has two ports, A and B. They are referred to as `tessel.port['A']`.
```

Comment:
All of the docs and examples, etc. for the Tessel 1 used square brackets, eg. `tessel.port['A']`, but these are unnecessary. While `tessel.port.A`, `tessel.port.B` are sufficient, you may opt to continue using the bracket notation if you'd prefer to maintain consistency—or change them all in one go.